signalfx/__init__.py

```diff
@@ -109,6 +108,9 @@ def ingest(self, token, endpoint=None, timeout=None):
             timeout=timeout or self._timeout)
 
     def signalflow(self, token, endpoint=None, timeout=None):
+
+        import signalflow
```

Comment:
This breaks the method's documentation. Move to just before the `return`. Also don't need the empty lines around it.